webpack/ForemanRhCloudHelpers.js

```diff
@@ -4,3 +4,7 @@
  * should be imported once core moves it to the ReactApp folder.
  */
 export const foremanUrl = path => `${window.URL_PREFIX}${path}`;
+
+export const isNotRhelHost = ({ hostDetails }) =>
```

Comment:
Might be nitpicking, but since the const is only called as `!isNotRhelHost`, I'd recommend changing it to `isRhelHost`.

Reply:
Notice that the `isNotRhelHost` function is also used in webpack/ForemanRhCloudFills.js, where it is passed to hide the tab.
It [needs to be a function](https://github.com/theforeman/foreman/blob/13bdb7f81a4fcc08eee82e726bd96c644aa23a05/webpack/assets/javascripts/react_app/components/HostDetails/index.js#L89-L92) and I preferred this over wrapping it in a lambda.